tests/python/unittest/test_gluon.py

```diff
@@ -1493,55 +1237,24 @@ def __init__(self, b1, b2):
     # Test default behavior
     c.save_parameters(param_path, deduplicate=False)
 
-    params = mx.nd.load(param_path)
+    params = mx.npx.load(param_path)
     assert len(params) == 2  # Only a single copy of the shared parameter is saved
 
     b1 = B()
     b2 = B().share_parameters(b1.collect_params())
     c = C(b1, b2)
     c.load_parameters(param_path)
 
-def test_symbol_block_save_load(tmpdir):
-    tmp = str(tmpdir)
-    tmpfile = os.path.join(tmp, 'resnet34_fp64')
```

Comment:
@leezu how shall we support the inference use case for exported graph? Should we keep the SymbolBlock?